index.js

```diff
@@ -18,6 +18,7 @@ export default function ({types: t}) {
   const processChildren = (children) => {
     return children.map(c => {
       if (t.isJSXElement(c) || t.isStringLiteral(c) || t.isJSXExpressionContainer(c)) {
+        if (t.isStringLiteral(c)) return t.JSXText(c.value);
```

Comment:
Do you want to separate these scenarios out a bit?

* Leave elements/containers intact
* Wrap strings in JSXText
* Wrap everything else in a container